### PR TITLE
User manual

### PR DESCRIPTION
**Pull request by wdower**
adding guidance course

### CODE REVIEW

File: courses/guidance/04.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+## 4. Starting The Project
+
+### 4.1
+- Now you can start your project, after outlining the needs of your application and choosing an SRG.
+
+- At the top of the page, click the "Start New Project" button. 
+
+![Alt text](../../assets/img/Vulcan_Main_Page.png)
+
+- Name your project, give it a description, and choose its visibility.
```
Comment (vanessuniq) [inline courses/guidance/04.md:17]: might be worth to know that we can add a slack channel at the project level on project creation (optional). Also, admin can configure/change the visibility of a project after creation.
Not sure if necessary, but maybe touch up on private vs discoverable project and highlight the feature that users not part of a project can request to join a discoverable project.

File: courses/guidance/05.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+author: Sumaa Sayed
+headerDepth: 3
+---
+
+## 5. Components
+
+### 5.1 What is a Component?
+[add text here]
+
+### 5.2 Creating a Component for Your Project
```
Comment (vanessuniq) [inline courses/guidance/05.md:13]: Maybe it is suited for another section, but might want to list different ways to create a new component ( copy from spreadsheet, copy component, duplicate) and the purpose for each.

File: courses/guidance/06.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+author: Sumaa Sayed
+headerDepth: 3
+---
+
+## 6. Tailoring Your Components
+
+### 6.1
+- For your application, you'll need to decide what controls will be appliable to your application, whihc ones need some tailoring, and which ones will not be applicable to your application.
+
+### 6.2 Editing Components
```
Comment (vanessuniq) [inline courses/guidance/06.md:13]: might be good to state that for each status, there are some fields that must be populated before saving the control.

File: courses/guidance/notes.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+This guide assumes that the user is logged into the website: https://mitre-vulcan-prod.herokuapp.com/ -- use the vulcan.mitre.org to write this out
+
+-use the readme: https://github.com/mitre/saf-training-lab-environment#readme
+-launch it locally (codespace training)
+- for any ss, ![Alt text](../../assets/img/SAF_Capabilities_Validate.png) -- add them to the assets folder, rename them so you know what the ss is
+- each step should be a separate page (for now)
+
+
+Other important things to note:
```
Comment (vanessuniq) [inline courses/guidance/notes.md:17]: touchup on the `related rules` feature and `rule satisfaction`

File: courses/guidance/04.md (hunk 1 above)
Comment (wdower) [reply courses/guidance/04.md:17]: Good point.

**Comment by wdower** [comment]
merging in favor of smaller, more targeted PRs after conversation with team

**Pull request merged**